src/query/graphite/native/aggregation_functions_test.go

```diff
@@ -150,6 +152,10 @@ func (e mockEngine) FetchByQuery(
 	return e.fn(ctx, query, opts)
 }
 
+func (e mockEngine) Storage() storage.Storage {
+	return e.storage
```

Comment:
nit: this is equivalent to `return nil`

Reply:
ok, i changed it to `return nil` and added a comment explaining why the function exists